### ISSUE TITLE
Does the repo lack the required source code for training?

### ISSUE THREAD

**Issue by vkgo** [lingochamp/Multi-Scale-BERT-AES#3]
It seems I can't find the training source codes in this repository.

**Comment by rahuls321** [comment]
Yes it's missing from the repo.

**Comment by iamhere1** [comment]
Hi, for some reasons, the complete training code is not convenient to be pushed now. However, we have published most of the hyper parameters in the [paper](https://arxiv.org/abs/2205.03835). If you want to train your AES model, here are some tips:
1.  Based on our open source code, run the decoding process to see if the QWK metric is consistent with the paper.
2. Add the training process (mainly include the process of updating the gradient according to the loss function. The code for data loading, data encoding,  and the model have been published in the repo).

**Comment by shield124** [comment]
> Hi, for some reasons, the complete training code is not convenient to be pushed now. However, we have published most of the hyper parameters in the [paper](https://arxiv.org/abs/2205.03835). If you want to train your AES model, here are some tips:
> 
> 1. Based on our open source code, run the decoding process to see if the QWK metric is consistent with the paper.
> 2. Add the training process (mainly include the process of updating the gradient according to the loss function. The code for data loading, data encoding,  and the model have been published in the repo).

作者大大，可以分享下train模型的代码吗？研一入门小白实在难以自己写出来训练的代码，感谢您！

**Comment by iamhere1** [comment]
> > Hi, for some reasons, the complete training code is not convenient to be pushed now. However, we have published most of the hyper parameters in the [paper](https://arxiv.org/abs/2205.03835). If you want to train your AES model, here are some tips:
> > 
> > 1. Based on our open source code, run the decoding process to see if the QWK metric is consistent with the paper.
> > 2. Add the training process (mainly include the process of updating the gradient according to the loss function. The code for data loading, data encoding,  and the model have been published in the repo).
> 
> 作者大大，可以分享下train模型的代码吗？研一入门小白实在难以自己写出来训练的代码，感谢您！

预测代码可以先跑下和理解下，训练部分你先尝试加下损失函数和梯度更新的逻辑，如果有问题，欢迎继续交流。
